website/prereg/utils.py

```diff
@@ -1,9 +1,33 @@
 from modularodm import Q
 
-def get_prereg_schema():
+from website.util import permissions
+
+
+PREREG_CAMPAIGNS = {
+    'prereg': 'Prereg Challenge',
+    'erpc': 'Election Research Preacceptance Challenge',
+}
+
+def drafts_for_user(user, campaign):
```

Comment:
This function could use unit tests. This can happen in a followup PR though. I'm writing some basic tests for `get_prereg_schema` at the moment, so you'll have a place to put tests for this function.